features/navigation_bar_left/leftNavigationBar.feature

```diff
@@ -24,19 +24,17 @@ Feature: Test set to check the left side menu items
         And <userRole> clicks left navigation item 'calendar'
```

Comment:
Comment for line 7: Rename scenario to make clear that admin and teacher are used in this TC
@teacherAdminClicksLeftMenuItems or similar (nonStudent)